CMakeLists.txt

```diff
@@ -1,16 +1,34 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.10)
 
+execute_process(
```

Comment:
can this be 'fetched'? i ask because i also copied this into my project and it would be cool to have this shared somehwere. just wondering since it's not a big deal to copy/paste it :)

Reply:
probably, but it would be almost same amount of code